deb/common/docker-ce.postinst

```diff
@@ -9,6 +9,9 @@ case "$1" in
 			fi
 		fi
 		;;
+	upgrade)
+		invoke-rc.d docker start || exit $?
```

Comment:
expected behaviour is to have docker service started after completion of install of a docker-ce deb package. this is what happens on a new install of a docker-ce deb package.